app/javascript/gobierto_plans/webapp/lib/router.js

```diff
@@ -22,42 +33,66 @@ export const router = new VueRouter({
       component: Main,
       children: [
         {
-          path: "/",
-          name: "home",
-          component: Home
-        },
-        {
-          path: "categoria/:id",
-          name: "categories",
-          component: Categories
-        },
-        {
-          path: "proyecto/:id",
-          name: "projects",
-          component: Projects
-        },
-        {
-          path: "tabla/:id",
-          component: Groups,
+          path: "",
+          component: PlanTab,
           children: [
             {
-              path: "/",
-              name: "table",
-              component: GroupsByTerm
+              path: "",
+              name: routes.PLAN,
+              component: Plan,
+              meta: {
+                tab: routes.PLAN,
+                button: routes.PLAN
+              }
             },
             {
-              path: ":term",
-              name: "term",
-              component: ProjectsByTerm
+              path: "categoria/:id",
+              name: routes.CATEGORIES,
+              component: Categories,
+              meta: {
+                tab: routes.PLAN,
+                button: routes.PLAN
+              }
             },
-          ]
+            {
+              path: "proyecto/:id",
+              name: routes.PROJECTS,
+              component: Projects,
+              meta: {
+                tab: routes.PLAN,
+                button: routes.PLAN
+              }
+            },
+            {
+              path: "tabla/:id",
+              component: Groups,
+              children: [
+                {
+                  path: "/",
+                  name: routes.TABLE,
+                  component: GroupsByTerm,
+                  meta: {
+                    tab: routes.PLAN
+                  }
+                },
+                {
+                  path: ":term",
+                  name: routes.TERM,
+                  component: ProjectsByTerm,
+                  meta: {
+                    tab: routes.PLAN
+                  }
+                },
+              ]
+            }
+          ],
         },
         {
-          path: "dashboards/:dashboardid?",
-          name: "dashboards",
-          component: Dashboards
+          path: "dashboards/:dashboardId?",
```

Comment:
Maybe it'd be necessary to document the list of steps to add dashboards to an existing Gobierto module?

1 - add routes to the vue component
2 - create a tab component
3 - add rails routes
...

Reply:
It'll depend on the module the dashboards are gonna be included. I mean, let's say a new module provides a route, `site.gobify.net/dummy/dashboars`, to render directly the dashboards. You won't need anything (but the rails routes) to make it work.
Vue-router and Tabs are specifications of the one single design, not general.